catblock/contentscript.js

```diff
@@ -440,7 +455,7 @@ var picinjection = {
                 de: "Keine Katzen mehr anzeigen!",
                 ru: "Не показывать кошек!",
                 nl: "Toon geen katten meer!",
-                zh: "不显示猫图片！",
+                zh: "不显示猫图片！"
             },
             "ok_no_more": {
                 en: "OK, AdBlock will not show you any more cats.\n\nHappy April Fools' Day!",
```

Comment:
@kpeckett I am not sure, but if that block of code is not used anymore, I would like to remove it in another PR.